modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/spring/ComponentConfigurationBuilder.java

```diff
@@ -481,6 +479,12 @@ public void onMultipleValues(KeyAttributeDefinitionPair[] definitions) {
     }
   }
 
+  private boolean shouldSetDefaultFrequencyForFixedFrequencyStrategy(String parameterName, ComponentParameterAst parameter) {
+    return "frequency".equals(parameterName)
+        && ownerComponent.getIdentifier().equals(FIXED_FREQUENCY_STRATEGY_IDENTIFIER)
+        && parameter.isDefaultValue() && parameter.getResolvedRawValue() == null;
```

Comment:
Shouldn't parameter.getResolvedRawValue() be different from `null`?

Reply:
It returns `null` when no frequency value is explicitly set on the` fixed-frequency `component